tasks/opendose-analysis.tex

```diff
@@ -0,0 +1,61 @@
+  The OpenDose collaboration was recently established to generate reference dosimetric data for a large number of computing models, using various Monte Carlo codes available in each participating team. The collaboration includes at the moment 14 research teams over 18 institutes. The idea is to impulse the collaborative development of a reference database, freely available, proposing dosimetric data applicable in a context of radiopharmaceutical dosimetry (for therapy and diagnostic applications). A major aspect is the development of  tools ensuring traceability and sustainability of generated results.
+
+  % Technical description
+  OpenDose data is produced using the five most represented Monte Carlo simulation softwares in medical applications: Geant4/GATE, MCNP, EGS, PENELOPE and Fluka. Each simulation consists of calculating radiation transport in anthropomorphic models  for specific parameters (source organ, particle type, energy, model and number of primaries to simulate).
+  Every simulation produces binary (3D matrices) and ASCII files for a total of ~150MB / simulation. The 3D matrices contain energy deposited per voxels, and ASCII files contain pre-processed data corresponding to energy deposited per regions such as organs and tissues. These raw outputs are later processed into dosimetric data such as Specific Absorbed Fractions (SAFs) and fed into a database.
+
+  Producing data for one model (ex. Adult female) requires $\sim 30 000$ simulations, with the work load shared between the different teams and softwares. 
+
+  This collaborative effort rise some challenges:
+  \begin{compactitem}
+  \item Amount of results: Terabytes of data that can be heterogeneous
+  \item Database: has to be efficient and handle all the results 
+  \item Data analysis: processing raw data into dosimetric data in a robust and reproducible way
+  \item Visualization: display and compare results from all teams
+  \end{compactitem}
+
+
+  The data produced by all the teams is currently centralised in CRCT, processed and fed into a local SQL database. 
+  The objective of this task is to build on the Jupyter ecosystem to create a unified data analysis framework for the members of the OpenDose collaboration.
+
+  \textbf{Jupyter workspace for OpenDose members}\\
+  For the OpenDose project members, Jupyter/Binder will provide the platform for all the tools to access, process and display results in a traceable and reproducible way. 
+  The task includes the following activities:
+  \begin{compactitem}
+  \item Developing tools to work seamlessly on the SQL database
+  \item Developing a common data analysis pipeline
+  \item Developing visualization tools
+  \item Comparing results between teams
+  \item Disseminating results
+    (\delivref{workpackage}{opendose-analysis})
```

Comment:
For deliverable references within a work package:

```suggestion
    (\localdelivref{opendose-analysis})
```

The `\delivref{workpackage-id}{deliv-id}` format is needed only for referencing a deliverable from another work package.

Reply:
I've updated the task template to use localdelivref, too